source/includes/steps-create-agg-pipeline.yaml

```diff
@@ -16,10 +16,11 @@ title: Fill in the pipeline stage.
 level: 4
 ref: fill-in-stage
 content: |
-   Fill in your selected stage. As you modify the pipeline stage, the
-   preview documents shown in the pane to the right of the stage
-   update automatically to reflect the results of your pipeline as it
-   progresses, provided :guilabel:`Auto Preview` is enabled:
+   Fill in your selected stage. If you enable :guilabel:`Auto Preview`,
```

Comment:
[suggestion]

Consider: "If you enable :guilabel:`Auto Preview`, preview documents will update automatically to the right of the stage to reflect the results of your pipeline as it progresses."

Reply:
Done